.circleci/config.yml

```diff
@@ -4,6 +4,7 @@ defaults: &defaults
     - image: circleci/ruby:2.5.1-node
       environment:
         - RAILS_ENV: test
+        - CC_TEST_REPORTER_ID: test_reporter_id
```

Comment:
You don't need this line, instead of it would be better to change your env variable to `CC_TEST_REPORTER_ID`.